### PR TITLE
Update README.md

### LINKED ISSUE #19: issue with readme.md
two items
1. Change the order. The additional topics should be last.
2.The title for the workflow - should be short and more to the point

Call it 

Burger Meal Order Workflow  or similar
something not more than 4-5 words

### PR DESCRIPTION
**Pull request by Aaron-Gor**
@lsnovich Fixed #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,17 +9,8 @@ The overall concept for this embryo API is to take restaurant orders from seated
...
-[POST Bugermeal Properties File](POST-Properties-Reference.md) Properties and descriptions for POST actions.
-
-[GET Properties File](GET-Properties-Reference.md) Properties and descriptions for GET actions (get bill).
-
-[WORKFLOW Diagram](AaronAPIWorkflow.png) Workflow diagram for burgermeal order.
-
-
 
-### Diagram Showing workflow for ordering a burgermeal 
+### Burgermeal Order Workflow
```
Comment (lsnovich) [inline README.md:13]:
```suggestion
### Burger Meal Order Workflow
```

**Review by lsnovich — CHANGES_REQUESTED** [review]
one small change and then merge

**Comment by Aaron-Gor** [comment]
@lsnovich Fixed #22

**Comment by Aaron-Gor** [comment]
@lsnovich Fixed #22

**Pull request merged**